### ISSUE TITLE
GN header dependency error: "spv-amd-shader-ballot.insts.inc"

### ISSUE THREAD

**Issue by null77** [KhronosGroup/SPIRV-Tools#2827]
This is breaking the roll into ANGLE. It seems to be a recent regression as of Aug 30/2019.

```ERROR at //third_party/spirv-tools/src/source/opt/amd_ext_to_khr.cpp:19:11: Can't include this header from here.
#include "spv-amd-shader-ballot.insts.inc"
          ^------------------------------
The target:
  //third_party/spirv-tools/src:spvtools_opt
is including a file from the target:
  //third_party/spirv-tools/src:spvtools_vendor_tables_spv-amd-shader-ballot

It's usually best to depend directly on the destination target.
In some cases, the destination target is considered a subcomponent
of an intermediate target. In this case, the intermediate target
should depend publicly on the destination to forward the ability
to include headers.

Dependency chain (there may also be others):
  //third_party/spirv-tools/src:spvtools_opt -->
  //third_party/spirv-tools/src:spvtools --[private]-->
  //third_party/spirv-tools/src:spvtools_vendor_tables_spv-amd-shader-ballot
```

See https://chromium-review.googlesource.com/c/angle/angle/+/1777214
and https://ci.chromium.org/p/angle/builders/try/win-clang-x64-dbg/7558

**Comment by null77** [comment]
@dneto0 fyi. It's not breaking the roll into Chromium right now. But I'd like to add the header check to Chromium to make it a bit more visible as well.

**Comment by null77** [comment]
Most likely: https://github.com/KhronosGroup/SPIRV-Tools/commit/35d98be3bc87e34da31de5f484c57272078d57e8

@s-perron fyi, can you help fix?

**Comment by null77** [comment]
n/m this was fixed by cwallez in #2821
